examples/src/Config/Movement/FreeOnDrag.elm

```diff
@@ -5,6 +5,7 @@ import DnDList
 import Html
 import Html.Attributes
 import Html.Events
+import Port exposing (onPointerMove, onPointerUp, releasePointerCapture)
```

Comment:
Please adjust the import to follow the coding conventions of this package by using fully qualified names instead of `exposing`. Change it to `import Port` and use `Port.onPointerMove`, `Port.onPointerUp`, etc., in the code. Thanks!

Reply:
Please adapt this for all files :smile:

Reply:
Ok, this is done.

Reply:
Great, thanks!